CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.14)
+cmake_minimum_required(VERSION 3.20)
```

Comment:
I've version 3.18.4 and it works, so I think the minimum version should be 3.18 at most (not sure what is the real minimum version though, it didn't work with 3.16.x, right?)

Reply:
Hey, yes 3.16 it didn't work.

if 3.18 works, shall do that :)

Reply:
done :)